introduction.md

```diff
@@ -0,0 +1 @@
+Hi I'm Mark
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.

Reply:
done!

Reply:
From GitHub Desktop:
Cloning into 'C:\Users\marshall.owens\Documents\GitHub\dojo_rules'...
fatal: unable to access 'https://github.com/OwensMT/dojo_rules.git/': schannel: next InitializeSecurityContext failed: Unknown error (0x80092012) - The revocation function was unable to check revocation for the certificate.
From GitBash:
$ git clone https://github.com/OwensMT/dojo_rules
Cloning into 'dojo_rules'...
fatal: unable to access 'https://github.com/OwensMT/dojo_rules/': SSL certificate problem: unable to get local issuer certificate